app-android/src/main/java/io/github/droidkaigi/confsched2022/MainActivity.kt

```diff
@@ -14,6 +16,13 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         WindowCompat.setDecorFitsSystemWindows(window, false)
+        window.statusBarColor = Color.TRANSPARENT
+        window.navigationBarColor =
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+                Color.TRANSPARENT
+            } else {
+                Color.argb((255 * 0.5).toInt(), 0, 0, 0)
```

Comment:
// Make it easy to see for older OS versions

Reply:
Thank you. I did it!